src/site/stages/build/plugins/create-react-pages.js

```diff
@@ -1,54 +1,34 @@
 /* eslint-disable no-param-reassign, no-continue, no-console */
 const path = require('path');
 const manifestHelpers = require('../webpack/manifest-helpers');
-const BUILD_TYPE = require('../../../constants/environments');
 
-const prodEnvironments = new Set([BUILD_TYPE.VAGOVPROD]);
-
-function createTemporaryReactPages(options) {
+function createReactPages() {
   return (files, metalsmith, done) => {
-    if (!prodEnvironments.has(options.buildtype)) {
-      const root = path.join(__dirname, '../../../../..');
-      manifestHelpers
-        .getAppManifests(root)
-        // Skip manifests that are mapping the url from settings.js
-        .filter(m => m.rootUrl)
-        .forEach(({ entryName, appName, rootUrl }) => {
-          // Removing leading and trailing slashes
-          const trimmedUrl = rootUrl.endsWith('/')
-            ? rootUrl.substring(1, rootUrl.length - 1)
-            : rootUrl.substring(1);
-          const filePath = `${trimmedUrl}/index.html`;
+    const root = path.join(__dirname, '../../../../..');
+    manifestHelpers
+      .getAppManifests(root)
+      // Skip manifests that are mapping the url from settings.js
+      .filter(m => m.rootUrl)
+      .forEach(({ entryName, appName, rootUrl, template }) => {
+        const trimmedUrl = path.join('.', rootUrl);
+        const filePath = path.join(trimmedUrl, 'index.html');
 
-          if (!files[filePath]) {
-            console.log(`Creating temp React landing page: ${rootUrl}`);
-            files[filePath] = {
-              title: appName,
-              entryname: entryName,
-              layout: 'page-react.html',
```

Comment:
It might make sense to keep this here. The `...template` below will override the `layout` if it's provided in the manifest.

Reply:
I actually kinda linked forcing each manifest to have that property, to possibly help demystify things a bit. But I also don't mind adding that back.

Reply:
Yeah, I mean I'm not strongly opinionated here. This may end up being just as mystical, though. :man_shrugging: